connector/spanmetricsconnector/README.md

```diff
@@ -130,7 +130,7 @@ The following is a simple example usage of the `spanmetrics` connector.
 
 For configuration examples on other use cases, please refer to [More Examples](#more-examples).
 
-The full list of settings exposed for this connector are documented [here](../../connector/spanmetricsconnector/config.go).
```

Comment:
I'm not convinced this is an improvement. Why include this in a spelling PR?

Reply:
The tooling happens to be able to identify problematic patterns/anti patterns and this happens to be a pattern that it can use.

The specific rule here is:
##### Do not use `(click) here` links

For more information, see:
* https://www.w3.org/QA/Tips/noClickHere
* https://webaim.org/techniques/hypertext/link_text
* https://granicus.com/blog/why-click-here-links-are-bad/
* https://heyoka.medium.com/dont-use-click-here-f32f445d1021
```
(?i)(?:>|\[)(?:(?:click |)here|link|(?:read |)more)(?:</|\]\()
```

---

I'm happy to omit such changes.

Reply:
Ah, I don't feel strongly about it but slightly prefer human language in links over file names.

Maybe an issue can propose this as a standard first? (While correct spelling requires no discussion.)

Reply:
If you have a proposal for text that's actually meaningful, I'd be more than happy to take it. I tried very hard in many places. I only punted in a couple when I couldn't come up with anything better than the file name.